packages/core/scripts/test/testE2E.ts

```diff
@@ -5,5 +5,9 @@
 
 import { runToolkitTests } from './launchTestUtilities'
 void (async () => {
-    await runToolkitTests('e2e', 'dist/src/testE2E/index.js')
+    const relativeEntrypoint = process.argv[2]
+    if (!relativeEntrypoint) {
+        throw new Error('Relative entrypoint is required')
```

Comment:
Is "relative" mentioned as in "relative path"? An example in the error would also be a good hint.

    throw new Error('relative path to entrypoint is required (example: dist/src/testE2E/index.js)')